lnschema_core/models.py

```diff
@@ -1536,8 +1534,8 @@ class Feature(Record, CanValidate, TracksRun, TracksUpdates):
 
         *Features* and *labels* denote two ways of using entities to organize data:
 
-        1. A feature qualifies *what* is measured, i.e..  numerical or categorical random variable
-        2. A label *is* a measured value, i.e..  category
+        1. A feature qualifies *what* is measured such as a numerical or categorical random variable
```

Comment:
This gotta be `i.e.` because it's a strict exhaustive definition ("such as" or "e.g." would imply non-exhaustiveness as you saw in your Claude prompt: https://claude.ai/share/5b2f9083-c64d-4817-bc5a-4ecef93ecb90)